### PR TITLE
show cmdline and cwd in procinfo

### PR DESCRIPTION
**Pull request by dmur1**

### CODE REVIEW

File: pwndbg/commands/procinfo.py [hunk 1]
```diff
@@ -211,6 +223,10 @@ def procinfo() -> None:
     if not proc.status:
         return
 
+    # print("%-10s %s" % ("cmdline", proc.something))
```
Comment (dmur1) [inline pwndbg/commands/procinfo.py]: for some reason when i try to use proc.cmdline i get an error message like this:

```
pwndbg> procinfo
exe        '/usr/bin/ls'
Python Exception <class 'FileNotFoundError'>: [Errno 2] No such file or directory: '/home/dan/pwndbg/pwndbg/../requirements.txt'
Error occurred in Python: [Errno 2] No such file or directory: '/home/dan/pwndbg/pwndbg/../requirements.txt'
```

( yes the comment shows proc.something my bad but i was testing with proc.cmdline )

**Comment by dmur1** [comment]
doesn't quite work due to an issue i was having getting the cmdline - maybe its cache related but i don't know how this works

File: pwndbg/commands/procinfo.py (hunk 1 above)
Comment (disconnect3d) [reply pwndbg/commands/procinfo.py]: This comes from https://github.com/pwndbg/pwndbg/issues/1865 - we need to fix this ASAP :/

**Comment by dmur1** [comment]
using https://github.com/pwndbg/pwndbg/pull/1879 i was able to see the actual issue and fix it at least

**Pull request merged**